examples/spark/src/main/scala/org/apache/carbondata/examples/util/ExampleUtils.scala

```diff
@@ -81,15 +85,6 @@ object ExampleUtils {
         .map(x => ("a", "b", x))
         .toDF("c1", "c2", "c3")
 
-    // save dataframe to carbon file:(df->csv->carbon file)
-    df.write
-        .format("carbondata")
-        .option("tableName", tableName)
-        .option("compress", "true")
```

Comment:
This function is supposed to write only once. It is also used in `appendSampleCarbonFile`, if this function writes twice, it is wrong.